ui/src/wizard/AzureWizard.ts

```diff
@@ -52,6 +53,7 @@ export class AzureWizard<T> implements types.AzureWizard<T>, IInternalAzureWizar
 
                 actionContext.properties.lastStepAttempted = `prompt-${step.constructor.name}`;
                 this._title = step.wizardNode.effectiveTitle;
+                this.hideStepCount = step.hideStepCount;
```

Comment:
So this is a per-step setting?  Not sure that makes sense to me.

Reply:
I ran into this when users were creating a project or function. In both wizards, I first prompt for something that determines most of the rest of the wizard (project language or function template). The step count was showing as `1/1` even though it should be `1/?`. The only other option I see is to hide on a per-wizard basis, but that's not what I wanted in this scenario. It's still helpful for users to see how many steps there are _after_ they select a language/template. We could always add the per-wizard option later